src/snowcli/cli/streamlit/manager.py

```diff
@@ -38,6 +37,48 @@ def share(self, streamlit_name: str, to_role: str) -> SnowflakeCursor:
     def drop(self, streamlit_name: str) -> SnowflakeCursor:
         return self._execute_query(f"drop streamlit {streamlit_name}")
 
+    def _deploy_experimental(
+        self,
+        streamlit_name: str,
+        main_file: Path,
+        environment_file: Optional[Path] = None,
+        pages_dir: Optional[Path] = None,
+        stage_name: Optional[str] = None,
+        warehouse: Optional[str] = None,
```

Comment:
```suggestion
        query_warehouse: Optional[str] = None,
```
To communicate the difference between query warehouse and connection warehouse